### DISCUSSION TITLE
Any way of adding an UI date?

### DISCUSSION THREAD

**Discussion by brotherelric** [discord-py-ui/discord-ui#102] [Q&A] [answered]
Just like this:
![Screenshot_1](https://user-images.githubusercontent.com/66126306/139105827-974bbac3-d9b3-44f2-95cd-51675e07f8c3.png)

**Comment by kvsxxx** [comment, marked as answer]
That's a [markdown](https://discord.com/developers/docs/reference#message-formatting) feature of discord:

```py
# <t:TIMESTAMP>

from datetime import datetime
f"<t:{int(datetime.now().timestamp())}>" # add this to your emebd content
```

**Reply by brotherelric** [reply]
Really sorry! Thought that was something which was included in this module. Thanks~
